server/services/companies/companies.hooks.ts

```diff
@@ -46,18 +46,42 @@ const votedPartners = {
   },
 };
 
+const pointPartners = {
```

Comment:
We can also use Objection to do the join. Any reason not to here?

Reply:
Do you mean $eager loading? I thought that was between tables. I'm just grabbing User info from an array of userIds